src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -319,8 +319,11 @@ public ResponseEntity<MeasureSet> createCmsId(
   public ResponseEntity<MeasureSet> associateCmsId(
       Principal principal,
       @RequestParam(required = true, name = "qiCoreMeasureId") String qiCoreMeasureId,
-      @RequestParam(required = true, name = "qdmMeasureId") String qdmMeasureId) {
+      @RequestParam(required = true, name = "qdmMeasureId") String qdmMeasureId,
+      @RequestParam(required = true, defaultValue = "false", name = "copyMetaData")
+          boolean copyMetaData) {
```

Comment:
```suggestion
      @RequestParam String qdmMeasureId,
      @RequestParam(defaultValue = false) boolean copyMetaData) {) {
```

Reply:
as we are providing default value it makes sense to remove required. thank you, I'll remove it

Reply:
by default, parameters are required. Also, I don't think you need an alias as the property name and the alias is the same